run/views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.decorators import api_view
+from rest_framework.parsers import JSONParser
+from .serializers import RunSerializer
+from .models import Run
+from rest_framework.response import Response
+
+
+@api_view(http_method_names=['POST'])
```

Comment:
Use class-based views.

Shouldn't this method return  the id of successful runs?